docker/docker-compose-yugabyte.yml

```diff
@@ -0,0 +1,46 @@
+version: '2'
+
+# Local Yugabyte database, see https://docs.yugabyte.com/latest/deploy/docker/docker-compose/
+
+volumes:
+  yb-master-data-1:
+  yb-tserver-data-1:
+
+services:
+  yb-master:
+    image: yugabytedb/yugabyte:latest
+    container_name: yb-master-n1
+    volumes:
+      - yb-master-data-1:/mnt/master
+    command: [ "/home/yugabyte/bin/yb-master",
+               "--fs_data_dirs=/mnt/master",
+               "--master_addresses=yb-master-n1:7100",
+               "--rpc_bind_addresses=yb-master-n1:7100",
+               "--replication_factor=1"]
+    ports:
+      - "7000:7000"
+    environment:
+      SERVICE_7000_NAME: yb-master
+
+  yb-tserver:
+    image: yugabytedb/yugabyte:latest
+    container_name: yb-tserver-n1
+    volumes:
+      - yb-tserver-data-1:/mnt/tserver
+    command: [ "/home/yugabyte/bin/yb-tserver",
+               "--fs_data_dirs=/mnt/tserver",
+               "--start_pgsql_proxy",
+               "--rpc_bind_addresses=yb-tserver-n1:9100",
+               "--tserver_master_addrs=yb-master-n1:7100",
+               "--ysql_sequence_cache_minval=1"]
+    ports:
+      - "9042:9042"
+      - "5433:5433"
```

Comment:
running Yugabyte on a different port in case one would like to have both running locally